server/server.go

```diff
@@ -107,12 +109,32 @@ func (s *CSIDriverProviderServer) Mount(ctx context.Context, req *v1alpha1.Mount
 	failoverRegion := attrib[failoverRegionAttrib]
 	usePodIdentityStr := attrib[usePodIdentityAttrib]
 	preferredAddressType := attrib[preferredAddressTypeAttrib]
+	httpTimeoutStr := attrib[httpTimeoutAttrib]
 
 	// Validate preferred address type
 	if preferredAddressType != "ipv4" && preferredAddressType != "ipv6" && preferredAddressType != "auto" && preferredAddressType != "" {
 		return nil, fmt.Errorf("invalid preferred address type: %s", preferredAddressType)
 	}
 
+	// Parse and validate HTTP timeout
+	var httpTimeoutDuration time.Duration
+	if httpTimeoutStr != "" {
+		var err error
+		httpTimeoutDuration, err = time.ParseDuration(httpTimeoutStr)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse httpTimeout value '%s': %v", httpTimeoutStr, err)
+		}
+		if httpTimeoutDuration <= 0 {
+			return nil, fmt.Errorf("httpTimeout must be positive, got: %v", httpTimeoutDuration)
+		}
+		if httpTimeoutDuration > 30*time.Second {
+			klog.Warningf("httpTimeout value %v is unusually high, consider using a smaller value", httpTimeoutDuration)
+		}
+	} else {
+		// Default to 100ms for backward compatibility
```

Comment:
I'm nitpicking, but we default to 100ms. It's not (just) for backwards compatibility, it's because we should have a default so customers don't have to configure this if they don't have a need to.

Probably just trim the comment down to `Default to 100ms` :)